.eslintrc

```diff
@@ -37,13 +37,10 @@ overrides:
       react/no-unused-prop-types: 1
       require-jsdoc: 1
   - files:
-      - "packages/search-metadata-previews/**/*.js"
+      - "packages/replacement-variable-editor/**/*.js"
```

Comment:
This is a strange change. I think this indicates something went wrong with your merging of the feature branch into here? (did you not fetch before doing a local merge perhaps?)
Because the replacement-variable-editor linting rules were removed in the feature branch.

Reply:
Yes, that indeed comes from a merge conflict badly resolved